pkg/virt-handler/isolation/process_test.go

```diff
@@ -0,0 +1,134 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2021 Red Hat, Inc.
+ *
+ */
+
+package isolation
+
+import (
+	"github.com/mitchellh/go-ps"
+
+	. "github.com/onsi/ginkgo"
+	"github.com/onsi/ginkgo/extensions/table"
+	. "github.com/onsi/gomega"
+)
+
+var emptyProcessList = []ps.Process{}
+
+var _ = Describe("process", func() {
+	const proc1Andproc5Exec = "processA"
+	const proc2Andproc4Pid = 110
+	const nonExistPPid = 300
+
+	procStub1 := ProcessStub{ppid: 1, pid: 120, binary: proc1Andproc5Exec}
+	procStub2 := ProcessStub{ppid: proc2Andproc4Pid, pid: 2222, binary: "processB"}
+	procStub3 := ProcessStub{ppid: 1, pid: 110, binary: "processC"}
+	procStub4 := ProcessStub{ppid: proc2Andproc4Pid, pid: 3333, binary: "processD"}
+	procsStubList := []ProcessStub{procStub1, procStub3, procStub2, procStub4}
+	processes := convertToPsGoProcessList(procsStubList)
+
+	Context("find child processes", func() {
+		proc2AndProc4StubList := []ProcessStub{procStub2, procStub4}
+		proc2AndProc4 := convertToPsGoProcessList(proc2AndProc4StubList)
+
+		table.DescribeTable("should return the correct child processes of the given pid",
+			func(processes []ps.Process, ppid int, expectedProcesses []ps.Process) {
+				Expect(childProcesses(processes, ppid)).
+					To(ConsistOf(expectedProcesses))
+			},
+			table.Entry("given no input processes, there are no child processes",
+				emptyProcessList, nonExistPPid, emptyProcessList,
+			),
+			table.Entry("given process list and non-exist pid, should return no child processes",
+				processes, nonExistPPid, emptyProcessList,
+			),
+			table.Entry("given process list and pid where there are child processes of the given pid",
+				processes, proc2Andproc4Pid, proc2AndProc4,
+			),
+		)
+	})
+
+	Context("lookup process by executable", func() {
+		proc1 := convertToPsGoProcess(&procStub1)
+
+		procStub5 := ProcessStub{ppid: 100, pid: 220, binary: proc1Andproc5Exec}
+		procsListWithProcsRunningSameExec := convertToPsGoProcessList(append(procsStubList, procStub5))
```

Comment:
same

Reply:
I removed `proc1` and `procsListWithProcsRunningSameExec`, now their values are inline.
Also moved the conversion to the test function.

Having `procStub5` in its own variable is more readable instead of define it inline on the test function, isnt it?